packages/integration-tests/test/shared/constants.ts

```diff
@@ -0,0 +1 @@
+export const DEPOSIT_FEED_ADDR = '0xdeaddeaddeaddeaddeaddeaddeaddeaddead0001'
```

Comment:
We are going to need common js stuff soon, its going to be scattered all over the place again

Reply:
Yeah. I think we should do this once we've stabilized and moved everything back into the monorepo.